core/src/main/java/com/datastrato/gravitino/connector/BaseCatalog.java

```diff
@@ -169,12 +180,59 @@ public CatalogOperations ops() {
     return ops;
   }
 
+  public AuthorizationPlugin getAuthorizationPlugin() {
+    if (authorization == null) {
+      synchronized (this) {
+        if (authorization == null) {
+          authorization = createAuthorizationPluginInstance();
+        }
+      }
+    }
+    return authorization.plugin();
+  }
+
+  private BaseAuthorization<?> createAuthorizationPluginInstance() {
+    String authorizationProvider =
+        (String) catalogPropertiesMetadata().getOrDefault(conf, AUTHORIZATION_PROVIDER);
+    if (authorizationProvider == null) {
+      throw new IllegalArgumentException("Authorization provider is not set");
+    }
```

Comment:
I found that the default value is null, so this will always throw an exception, right?

Reply:
Besides, I think this should not be a default behavior, right, we can let users to not configure the authorization plugin.

Reply:
Fixed